script/validate-vet

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+# # We will eventually get to the point where packages should be the complete list
+# IFS=$'\n'
+# files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^Godeps/' || true) )
+# unset IFS
+packages=(
+    app/cli/*.go
+    version/*.go
```

Comment:
nvm there is not so much to vet, so I'll do it real quick in this PR too :sweat_smile: